src/footer.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function footer() {
+  return <FooterStyled></FooterStyled>
+}
+
+const FooterStyled = styled.section`
```

Comment:
Aktuell glaube ich benötigst du den Footer noch nicht wirklich oder? Ich nehme an der ist schonmal Platzhalter für spätere Stories. Vielleicht wenn es wirklich ein footer werden soll auch das html element footer benutzen anstelle von section?

Reply:
Ja der Footer kommt später. Besten Dank für den Hinweis. Das macht Sinn. Die Section werde ich in Footer ändern.